manager/director/apps/sites/forms.py

```diff
@@ -65,6 +67,16 @@ def clean(self) -> Dict[str, Any]:
         ):
             self.add_error("name", "Project site names cannot start with a number")
 
+        if len(cleaned_data["users"]) == 0:
+            self.add_error("users", "You must select at least one user for this site")
+
+        if (
+            "users" in cleaned_data.keys()
+            and self.user not in cleaned_data["users"]
```

Comment:
Have you tested this both ways (i.e. succeeds if you add yourself, fails if you don't) as a non-admin?

Reply:
Yes. I have tested, as a non-admin, adding myself only, adding myself and someone else, adding only someone else, and adding nobody and they work fine (the last two fail); and the same as an admin (only the last one fails).